docs/site/BelongsTo-relation.md

```diff
@@ -75,18 +75,32 @@ export interface OrderRelations {
 export type OrderWithRelations = Order & OrderRelations;
 ```
 
+
 The definition of the `belongsTo` relation is inferred by using the `@belongsTo`
 decorator. The decorator takes in a function resolving the target model class
 constructor and designates the relation type. It also calls `property()` to
 ensure that the decorated property is correctly defined.
+Usage of belongsTo relation is similar to legacy datasource juggler.
+
+The `@belongsTo` decorator takes three parameters
+  - the target model Class,
+  - the relation definition has three attributes - keyFrom, keyTo, name
+    - keyFrom is a property name on the "source" model (e.g. Order.customerId). keyFrom attribute has become a dummy parameter and is not relevant any more with the recent changes to use the decorated target as the actual keyFrom value.
```

Comment:
maybe we can wrap the keyFrom using backticks? i.e. `keyFrom`.  Similarly for the other 2 parameter names.

Reply:
agreed